### PR TITLE
Create stuff

### PR DESCRIPTION
**Pull request by harmw**
Added some minimalistic API functions to be able to add/delete documents, add labels and create projects.

Also ran it through `flake8` :)

### CODE REVIEW

**Comment by afparsons** [comment]
Hi @harmw,

Thanks for the contribution! This is a fair bit of code to look over, so I'll try to get to it soon. Ping me if I don't do so in the next week.

On code style:

> Also ran it through flake8 :)

Hmm, I guess it is time to formally decide on this repository's code style.
As I noted in the original README.md (which I need to update now), I'm personally a fan of [Python Black](https://github.com/psf/black).
However, this repository really should use whatever formatting style the other doccano repositories use, whether that's black, flake8, or something else.

@Hironsan: do you have thoughts on code formatting? What does the main repository use?

File: doccano_api_client/__init__.py [hunk 1]
```diff
@@ -33,46 +11,84 @@ class _Router:
...
+
         Args:
             url_parameter (dict): Every value must be a list.
-        
+
         Returns:
             A URL parameter string. Ex: `?key1=u1&key1=u2&key2=v1&...`
         """
-        return ''.join(['?', '&'.join(['&'.join(['='.join([tup[0], str(value)]) for value in tup[1]]) for tup in url_parameter.items()])])
+        return ''.join(['?', '&'.join(['&'.join(['='.join(
```
Comment (Hironsan) [inline doccano_api_client/__init__.py:87]: Consider to using `urllib.parse.urlencode`:

```python
>>> import urllib.parse
>>> d = {'key1': 'value1', 'key2': 'value2'}
>>> urllib.parse.urlencode(d)
'key1=value1&key2=value2'
```

Comment (harmw) [reply doccano_api_client/__init__.py:87]: Actually, this part has gone untouched (apart from the _flake8_ commit) while I did consider changing it to use `requests` `params={}`. I was saving that for some other PR, to go in and fix/change current code and keep focus here around adding new code :)

**Comment by harmw** [comment]
@afparsons thanks! Only stuff got added I actually needed so it's not even complete yet and will see some future additions (likely in a next PR). However, this should be enough to get people going :)

**Comment by Hironsan** [comment]
In [doccano-transformer](https://github.com/doccano/doccano-transformer), GitHub Actions are used to check to code conventions. I think if we add the action to this repository, it will be better. But this is another issue.

**Pull request merged**